nixos/modules/services/backup/znapzend.nix

```diff
@@ -315,6 +315,14 @@ in
         '';
       };
 
+      mailErrorSummaryTo = mkOption {
+        type = str;
+        default = "";
+        description = lib.mdDoc ''
```

Comment:
```suggestion
        description = ''
```

Reply:
There's another pending pull request for znapzend which adds 'lib.mdDoc' to all description tags. However, it's a realy old one... So I rather stick to the way it's currently done 😄